test/functional/datastoresrp/resources/simulated_environment_test.go

```diff
@@ -67,7 +67,7 @@ func Test_Deployment_SimulatedEnv_BicepRecipe(t *testing.T) {
 			SkipObjectValidation:                   true,
 			PostStepVerify: func(ctx context.Context, t *testing.T, ct shared.RPTest) {
 				// Get pods in app namespace
-				label := fmt.Sprintf("radius.dev/application=%s", appName)
+				label := fmt.Sprintf("radapp.io/application=%s", appName)
```

Comment:
Do you know how this test is passing? If this label is wrong then it wouldn't match anything.

Reply:
Ah, I see that this test verifies that no pods were created.